pkg/webhook/github.go

```diff
@@ -659,3 +697,41 @@ func validateSignature(signature, secretKey string, payload []byte) error {
 	}
 	return nil
 }
+
+func getTitlesFromPushEvent(pe *github.PushEvent) (string, string) {
```

Comment:
An idea would be to have a single `getTitleFromEvent` that would do the switch statement on the event type - this way, adding support for getting the titles from a new event would mean only adding a case, as opposed to a new function.

WDYT?
I don't have a strong preference, so feel free to keep the current implementation if you think it makes more sense.

Reply:
I see two sides to it. What you've proposed would be tidier to look at and easier to maintain, so that's a plus. On the flip side, we also would have _already_ done a type switch in the calling function, passed the event as an `interface{}` and then we'd be doing _another_ type switch. The performance hit of that is probably negligible, but that otherwise avoidable second type switch is a little irksome, plus we'd lose out on some degree of type safety and have to deal with the case where the "event" we're passed isn't an event after all.

> I don't have a strong preference, so feel free to keep the current implementation if you think it makes more sense.

Since I see both sides.......... laziness wins. 🤷‍♂

Reply:
I fully agree with this statement.